my_enumerable.rb

```diff
@@ -0,0 +1,18 @@
+# MyEnumerable module implementing methods all, any and filter
```

Comment:
![Screenshot from 2022-10-27 11-22-23](https://user-images.githubusercontent.com/65068771/198246659-1a51f0ae-a78a-49b8-9e32-9ee0dd1dc704.png)

- [ ] As the previous reviewer said, kindly add a readme file to your project and customize it accordingly. You may use this [readme template](https://github.com/microverseinc/readme-template).